zowe/zowe-cli
# [v3] Remove remaining references to V1 profiles

**What It Does**

Resolves #2231

Some classes have been deprecated, not yet removed. We should consider removing them in a future PR before V3 GA.

**How to Test**

Run CLI commands that load arguments from profiles to verify that `CommandProcessor` behavior has not broken.

**Review Checklist**
I certify that I have:


**Additional Comments**
